README.md

```diff
@@ -6,7 +6,13 @@ You can see/use a NANO and BANANO instance of this bot on the [official banano d
 
 ## Features
 
-- `balance` : Display balance of your account
+
+## Commands
+#### For Graham(Nano Edition), replace . with ?
+
```

Comment:
I just don't think this needs to be in the documentation because you could run it in banano mode and use ? , it's just an option